src/expressionEditor.js

```diff
@@ -410,7 +412,8 @@ define([
                 tag: tag,
                 tagArgs: tagArgs,
             }));
-            editorContent.empty().append($xpathUI);
+            editorContent.html($xpathUI);
```

Comment:
[`html` is eqivalent to `empty().append`](http://stackoverflow.com/a/1903090/10840)